tests/test_routing.py

```diff
@@ -89,6 +89,14 @@ def test_405():
     }
 
 
+def test_found_no_slash():
+    response = client.get('/found')
+    assert response.status_code == 200
```

Comment:
It'd probably make more sense for us to check the result of `client.get('/found', follow_redirects=False)`, and ensure that a 302 is returned, with the correct headers.